### PR TITLE
🎨 e2e: update Kember test

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR teaches puppeteer how to go through the new kember test based on the voila viewer and app mode.

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/wrapper/Three.js [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
 
 /**
-  * ignore THREE
   * eslint new-cap: [2, {capIsNewExceptions: ["THREE", "Blob"]}]
   * eslint no-underscore-dangle: ["error", { "allowAfterThis": true, "enforceInMethodNames": true }]
   */
 
 /**
   * @asset(threejs/*)
+  * @ignore(THREE)
```
Comment (sanderegg) [inline services/static-webserver/client/source/class/osparc/wrapper/Three.js:27]: this will remove these warnings in devel mode? thanks!

**Review by sanderegg — APPROVED** [review]
thanks!

File: services/static-webserver/client/source/class/osparc/wrapper/Three.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/wrapper/Three.js:27]: Kenau!

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by mrnicegyu11 — APPROVED** [review]
thanks!

**Pull request merged**